capdl-loader-app/src/main.c

```diff
@@ -1024,8 +1024,10 @@ init_tcb(CDL_Model *spec, CDL_ObjID tcb)
 
     ZF_LOGF_IFERR(error, "");
 
+#ifndef CONFIG_KERNEL_RT
```

Comment:
Hi Chris, we now use `CONFIG_KERNEL_MCS`. Can you please update.

Reply:
Sorry, the loader has changed a lot since this PR. This won't be needed anymore.